sdk/workloads/Azure.ResourceManager.Workloads/tests/Tests/SviCRUDTests.cs

```diff
@@ -0,0 +1,268 @@
+                Thread.Sleep(GetOpsIntervalinMillis);
+                sviObjectstatus = lro.Value.GetSapVirtualInstance("A13").Value.Data.Status;
+                count++;
+            }
+            while (sviObjectstatus != SapVirtualInstanceStatus.Offline &&
+                count < 6);
+            Assert.Equals(lro.Value.GetSapVirtualInstance("A13").Value.Data.Status, SapVirtualInstanceStatus.Offline);
+            lro.Value.GetSapVirtualInstance("A13").Value.Stop(WaitUntil.Completed);
+            count = 0;
+            do
+            {
+                Thread.Sleep(GetOpsIntervalinMillis);
+                sviObjectstatus = lro.Value.GetSapVirtualInstance("A13").Value.Data.Status;
+                count++;
+            }
+            while (sviObjectstatus != SapVirtualInstanceStatus.Running &&
+                    count < 6);
+            Assert.Equals(lro.Value.GetSapVirtualInstance("A13").Value.Data.Status, SapVirtualInstanceStatus.Running);
+        }
+
+        /// <summary>
+        /// Gets an object of SAP Workload Resource for Single Server that should be used for PUT call.
+        /// </summary>
+        /// <param name="infraRgName"> Contains the infra RG
+        /// used as app resource group. </param>
+        /// <returns>PHP Workload Resource.</returns>
+        private SapVirtualInstanceData GetSingleServerPayloadToPut(string infraRgName, bool isInstall)
+        {
+            var testSapWorkloadJson = File.ReadAllText(
+                @"C:\Users\skottukkal\azure-sdk-for-net\sdk\workloads\Azure.ResourceManager.Workloads\tests\SingleServerInstall.json");
+            //var testSapWorkload =
+            //    JsonConvert.DeserializeObject<SapVirtualInstanceData>(testSapWorkloadJson);
+
+            JObject jObject = JsonConvert.DeserializeObject(testSapWorkloadJson) as JObject;
+
+            var deploymentConfiguration =
+                JsonConvert.DeserializeObject<DeploymentConfiguration>(
+                    jObject.SelectToken("properties.configuration").ToString());
+
+            var singleServerConfiguration = JsonConvert.DeserializeObject<SingleServerConfiguration>(
+                jObject.SelectToken("properties.configuration.infrastructureConfiguration").ToString());
+            singleServerConfiguration.AppResourceGroup = infraRgName;
+
+            var ssh = JsonConvert.DeserializeObject<SshConfiguration>(jObject.SelectToken("properties.configuration.infrastructureConfiguration.virtualMachineConfiguration.osProfile.osConfiguration.ssh").ToString());
+            var osConfiguration = JsonConvert.DeserializeObject<LinuxConfiguration>(
+                jObject.SelectToken(
+                    "properties.configuration.infrastructureConfiguration." +
+                    "virtualMachineConfiguration.osProfile.osConfiguration").ToString());
+
+            osConfiguration.Ssh = ssh;
+            singleServerConfiguration.VirtualMachineConfiguration.OSProfile.OSConfiguration = osConfiguration;
```

Comment:
shou;d'nt this already be present in json payload? why do we need to get multiple variables and re-assign them?

From what i understand, we just need to change softwareConfiguration based on isInstall and put configuration as it is in testSapWorkload object